app/propsGenerators/indexPagePropsGenerator.ts

```diff
@@ -65,13 +68,24 @@ function generateAllDhbsVaccineDoseData(
     if (southIslandDhbNames.includes(dhb.dhbName)) {
       regions.push("southIsland");
     }
+
+    const yesterdayDoseData = yesterdayVaccineData.find(
+      (yesterdayDhb) => yesterdayDhb.dhbName === dhb.dhbName
+    );
+
     return {
       ...dhb,
       regionIds: regions,
       hasMetFirstDoseTarget:
         dhb.firstDosesPercentage >= CONSTANTS.firstDoseTargetPercentage,
       hasMetSecondDoseTarget:
         dhb.secondDosesPercentage >= CONSTANTS.secondDoseTargetPercentage,
+      firstDosesChange: yesterdayDoseData
+        ? dhb.firstDosesTo90Percent - yesterdayDoseData?.firstDosesTo90Percent
+        : 0,
```

Comment:
❓ - Should these default to `null` ? If we ever had a historical view, 0 wouldn't be valid for the first day 🤔 

OMG we should do a historical view 👁️

Reply:
Made it null if there is not data for yesterday. Not sure fi have done the best null checks lol